docs/install/_linux.rst

```diff
@@ -32,6 +32,42 @@ Install from PPA repository
     sudo apt-get update
     sudo apt-get install kolibri
 
+Uninstall
+*********
+From the command line: ``sudo apt-get remove kolibri``.
+
+Upgrade
+*******
+
+When you use the PPA installation method, upgrades to newer versions will be automatic, provided there is internet access available.
+
```

Comment:
@jredrejo @benjaoming Do you think this location is the best for documenting ``kolibri-server``? it is a bit _hidden_ in the TOC, which might not be bag, given that it is still a pre-release...

Another thing to decide is if we want to add more detailed instructions as requested in #79.

Reply:
Great point, yes, it will end up too hidden. As I might have mentioned before, I would like if we worked on a tutorial similar to the Raspberry Pi tutorial, perhaps unifying some concepts later on. I think the tutorial could be called "Class-room servers". _That's only for the long run_.

I think that same title could work well in the install sub-TOC directly below all the other items. It will have to mention loud and clear that it's only available on Debian/Ubuntu.

![image](https://user-images.githubusercontent.com/374612/57362867-011de600-7180-11e9-93e8-ca2f61cb80a7.png)

I'd suggest to create a new file `install/_server.rst` ?

```
Classroom servers
-----------------

.. tip: Using ``kolibri-server`` on a CPU with 4 cores means serving up to 4x as many users as you would on a regular Kolibri. This is because it adds a configuration to the system that runs many Kolibri workers in parallel.

Kolibri can be optimized further, and we recommend to do so. **This method is only available for Debian/Ubuntu** platforms, and you can find slightly more elaborated instructions for Raspberrry Pi [link].

As an addition to the main Kolibri packages, you can install ``kolibri-server``, a package for Debian/ubuntu systems that contain the configurations necessary to serve Kolibri in an optimized fashion for CPUs with several cores.

.. warming: This is a new distribution package introduced
```